web/rebuild.go

```diff
@@ -62,6 +62,7 @@ func (s *QuickFeedService) rebuildSubmission(request *qf.RebuildRequest) (*qf.Su
 	}
 	// If we fail to get owners, we ignore sending on the stream.
 	if userIDs, err := runData.GetOwners(s.db); err == nil {
+		// TODO: Check how grades are sent to students.
```

Comment:
Should do it and remove it ;-)

Reply:
Added comment that sending the assignment as-is will include grades for all users participating in a group submission